src/main/java/org/opensearch/security/dlic/rest/support/Utils.java

```diff
@@ -224,22 +224,7 @@ public static String hash(final char[] clearTextPassword) {
      * @return new list of API routes prefixed with _opendistro... and _plugins...
      *Total number of routes is expanded as twice as the number of routes passed in
```

Comment:
There is no more expansion, it is a list of replaced routes

Reply:
fixed